runtime/onert/backend/train/MemoryPlanner.test.cc

```diff
@@ -276,4 +275,189 @@ TEST(WICPlanner, disposable_claim_release_test)
   });
 }
 
-// Add Testcase for LayerScopeTensorIndex, using PlannerVerifier
+TEST(BumpPlanner, layerscope_claim_test)
+{
+  PlannerVerifier<BumpPlanner, LayerScopeTensorIndex> p;
+
+  ASSERT_NO_FATAL_FAILURE({
+    p.claim(0, 0, 10, 0);
+    p.claim(1, 0, 20, 10);
+    p.claim(2, 2, 30, 30);
+    p.release(0, 0);
+    p.capacity(60);
+  });
+}
+
+TEST(FirstFitPlanner, layerscope_claim_release_test)
+{
+  PlannerVerifier<FirstFitPlanner, LayerScopeTensorIndex> p;
+
+  ASSERT_NO_FATAL_FAILURE({
+    // 0 CLAIM - 10
+    p.claim(0, 0, 10, 0);
+
+    // 1 CLAIM - 20
+    p.claim(1, 0, 20, 10);
+
+    // 2 CLAIM - 30
+    p.claim(2, 2, 30, 30);
+
+    // 0 RELEASE - 10
+    p.release(0, 0);
+
+    // 3 CLAIM - 20
+    p.claim(3, 1, 20, 60);
+
+    // 4 CLAIM - 5
+    p.claim(4, 1, 5, 0);
+
+    // 5 CLAIM - 10
+    p.claim(5, 1, 10, 80);
+
+    // 6 CLAIM - 5
+    p.claim(6, 1, 5, 5);
+
+    // 2 RELEASE - 30
+    p.release(2, 2);
+
+    // 7 CLAIM - 35
+    p.claim(7, 1, 35, 90);
+
+    // 8 CLAIM - 10
+    p.claim(8, 1, 10, 30);
+
+    // 4 RELEASE - 5
+    p.release(4, 1);
+
+    // 9 CLAIM - 10
+    p.claim(9, 0, 10, 40);
+
+    // 10 CLAIM - 10
+    p.claim(10, 0, 10, 50);
+
+    // 6 RELEASE
+    p.release(6, 1);
+
+    // 1 RELEASE
+    p.release(1, 0);
+
+    // 8 RELEASE
+    p.release(8, 1);
+
+    // 9 RELEASE
+    p.release(9, 0);
+
+    // 10 RELEASE
+    p.release(10, 0);
+
+    // 3 RELEASE
+    p.release(3, 1);
+
+    // 5 RELEASE
+    p.release(5, 1);
+
+    // 7 RELEASE
+    p.release(7, 1);
+
+    // CAPACITY - 125
+    p.capacity(125);
+  });
+}
+
+TEST(FirstFitPlanner, layerscope_neg_release_non_existing_index)
```

Comment:
Negative test names must begin with `neg_` to recognize whether they are negative tests.

Reply:
Could you rename the hidden negative tests above in another PR?

Reply:
Yes, of course. I will fix it now.  :)